### PR TITLE
Add js to vr

### PR DESCRIPTION
**Pull request by jackdavidweber**
Finally got js functionality to work in aframe. This will allow me to potentially incorporate the api into virtual reality which would be cool! Also added the big blocks to both sides with my initials which I thought was cool and it has the added benefit of hiding the clouds before they stop moving and after they disapear

image

image

[Here is a video demo](https://streamable.com/9ftnj8) of the site from top to bottom.

Note that the automatic merge issue is because I had to move script tags for aframe before my earlier pull request was reviewed, but should be very easy to resolve.

### CODE REVIEW

File: portfolio/src/main/webapp/template.html [hunk 1]
```diff
@@ -27,7 +27,58 @@
     
     <!-- Theme CSS -->  
     <link id="theme-style" rel="stylesheet" href="assets/css/styles.css">
+    <script src="https://aframe.io/releases/1.0.4/aframe.min.js"></script>
+    <script src="https://unpkg.com/aframe-environment-component@1.1.0/dist/aframe-environment-component.min.js"></script>
+
+
+    <script>
+      function getRandomArbitrary(min, max){
```
Comment (Danish21) [inline portfolio/src/main/webapp/template.html]: Nit:
Arbitrary in getRandomArbitrary does not seem to be adding much to the name. Maybe getRandomBetween?

File: portfolio/src/main/webapp/template.html [hunk 2]
```diff
@@ -27,7 +27,58 @@
...
+            sceneEl.appendChild(boxE2);
+            
+            var textEl = document.createElement('a-text');
+            textEl.setAttribute('text', {value: interestsArr[i]} )
+            textEl.setAttribute('text', 'color: black; width: 8; align: center')
+            textEl.setAttribute('position', {x: xpos, y: ypos-.2, z: zpos+1.5});
+            textEl.setAttribute('animation', 'property: object3D.position.x; to:35; dur:50000; easing: linear; loop:true');
+            textEl.setAttribute('animation', {delay: animationDelay})
+            sceneEl.appendChild(textEl);
+
```
Comment (Danish21) [inline portfolio/src/main/webapp/template.html]: Remove blank lines.

File: portfolio/src/main/webapp/template.html [hunk 3]
```diff
@@ -27,7 +27,58 @@
...
+              'Tree Climbing',
+              '3D Printing',
+              'Virtual Reality',
+            ]
+          
+          for (var i = 0; i < 7; i++) {
+            var xpos=-20;
+            var ypos= getRandomArbitrary(1.6, 10);
+            var zpos= getRandomArbitrary(-8,-14);
+            var animationDelay= 10000*i
```
Comment (Danish21) [inline portfolio/src/main/webapp/template.html]: Uber Nit:
Missing semicolon.

File: portfolio/src/main/webapp/template.html [hunk 4]
```diff
@@ -27,7 +27,58 @@
...
+              'Frisbee',
+              'Periodic Table',
+              'Name of the Wind',
+              'Rock Climbing',
+              'Tree Climbing',
+              '3D Printing',
+              'Virtual Reality',
+            ]
+          
+          for (var i = 0; i < 7; i++) {
```
Comment (Danish21) [inline portfolio/src/main/webapp/template.html]: Instead of 7 I would use interestsArr.length

File: portfolio/src/main/webapp/template.html [hunk 5]
```diff
@@ -27,7 +27,58 @@
...
+    <script src="https://aframe.io/releases/1.0.4/aframe.min.js"></script>
+    <script src="https://unpkg.com/aframe-environment-component@1.1.0/dist/aframe-environment-component.min.js"></script>
+
+
+    <script>
+      function getRandomArbitrary(min, max){
+          return Math.random()*(max-min) + min;
+      }  
     
+      AFRAME.registerComponent('interests', {
```
Comment (Danish21) [inline portfolio/src/main/webapp/template.html]: Nice job figuring out the JS API.

**Review by Danish21 — APPROVED** [review]
LGTM aside from some small comments.

**Comment by jackdavidweber** [comment]
Will fix all of the comments by Danish21 on Monday!

**Pull request merged**